src/main/java/com/orange/main/price/repo/PriceRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.orange.main.price.repo;
+
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+import com.orange.main.price.bo.Price;
+
+public interface PriceRepository  extends PagingAndSortingRepository<Price, Long>{ 
```

Comment:
style: extra and missing space

```suggestion
public interface PriceRepository extends PagingAndSortingRepository<Price, Long> { 
```